lua/entities/gmod_wire_textentry.lua

```diff
@@ -40,6 +40,7 @@ if CLIENT then
 			end,
 			"Enter","Cancel"
 		)
+		panel:SetBackgroundBlur(false)
```

Comment:
We have [Derma_StringRequestNoBlur](https://github.com/wiremod/wire/blob/4d4cc55c29788f0b8f06230d98a9b031c6101590/lua/wire/client/cl_wirelib.lua#L221C1-L221C1) for this.